### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by Shreya12125**
Updated the index.html by adding the Default profile picture for all users, Username, date, and time of posting Star ratings and review text

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,7 +9,45 @@
...
+  }
+  .profile-pic{
+    width: 60px;
+    height: 60px;
+    border-radius: 50%;
+  }
+  .review-content{
+    text-align: left;
+  }
+</style>
```
Comment (amri-tah) [inline index.html:50]: Please move this to its corresponding CSS file

File: index.html [hunk 2]
```diff
@@ -92,7 +130,40 @@ <h3>We put recipes to the <br>test, so you can<br>savor the success!</h3>
...
+        <div class="review-content">
+          <h3>${review.username}</h3>
+          <p>Posted on: ${review.date}</p>
+          <p>${'⭐'.repeat(review.stars)}</p>
+          <p>${review.review}</p>
+        </div>
+      `;
+      reviewList.appendChild(reviewDiv);
+    });
+  }
```
Comment (amri-tah) [inline index.html:163]: This needs to be in the individual recipe page, and not the landing page. The JavaScript code should be in its JS file as well.

**Review by amri-tah — CHANGES_REQUESTED** [review]
@Shreya12125 Please make these changes, and attach images of the changes once its done in this PR thread!

**Comment by amri-tah** [comment]
@Shreya12125 Please attach the images of the changes you made in the website in this PR thread

**Comment by Shreya12125** [comment]
Image of the output
image

File: index.html (hunk 1 above)
Comment (amri-tah) [reply index.html:50]: This is still not fixed, please remove it from the index.html file to CSS files/recipepage.css

File: index.html (hunk 2 above)
Comment (amri-tah) [reply index.html:163]: This too, 
image

This is not supposed to get displayed in the landing page

File: index.html [hunk 3]
```diff
@@ -92,7 +130,40 @@ <h3>We put recipes to the <br>test, so you can<br>savor the success!</h3>
...
+  <h1>Reviews</h1>
+  <div id="review-list"></div>
+</div>
+
+<script>
+  const reviews = [
+    { username: "Username1", date: "2024-12-12 13:30", stars: 5, review: "Amazing recipes! I tried the Burnt Rice Bowl and it was
+    { username: "Username2", date: "2024-11-24 14:45", stars: 5, review: "Loved it! Perfect and easy for beginners. " },
+    { username: "Username3", date: "2023-09-22 20:15", stars: 4, review: "Great unique recipes but a bit hard to follow" }
+  ];
```
Comment (amri-tah) [inline index.html:144]: This data needs to be read from the recipes.json file for each individual recipe

**Comment by amri-tah** [comment]
> Image of the output image


Could also try to improve the UI a bit

**Comment by amri-tah** [comment]
Hey @Shreya12125, thanks for your effort on this! I feel this issue needs a bit more exploration, so I’ll have to reassign it to someone else to ensure we get the best solution. Hope that’s okay!
